### PR TITLE
Sends captions through the stream

### PR DESCRIPTION
**Pull request by RuiOrey**
Also fixes some windows compilation problems
code adapted from branch `cea-608-json`

### CODE REVIEW

File: plugins/CMakeLists.txt [hunk 1]
```diff
@@ -25,7 +25,7 @@ if(WIN32)
 	if (MSVC)
 		add_subdirectory(win-ivcam)
 	endif()
-	add_subdirectory(obs-vst3)
+#	add_subdirectory(obs-vst3)
```
Comment (roddylindsay) [inline plugins/CMakeLists.txt:28]: does this impact non-win?

File: libobs/obs-output.c [hunk 2]
```diff
@@ -1269,20 +1275,15 @@ static inline void send_interleaved(struct obs_output *output)
...
 
-		double frame_timestamp =
-			(out.pts * out.timebase_num) / (double)out.timebase_den;
+		double frame_timestamp = (out.pts * out.timebase_num) / (double)out.timebase_den;
 
-		if (output->caption_head &&
-		    output->caption_timestamp <= frame_timestamp) {
-			blog(LOG_DEBUG, "Sending caption: %f \"%s\"",
-			     frame_timestamp, &output->caption_head->text[0]);
+		double duration = 0.033;
```
Comment (roddylindsay) [inline libobs/obs-output.c:1280]: is this the only change from the branch that I created before?

File: libobs/obs-encoder.c [hunk 3]
```diff
@@ -781,6 +781,8 @@ void obs_encoder_set_video(obs_encoder_t *encoder, video_t *video)
 
 	voi = video_output_get_info(video);
 
+	blog(LOG_INFO, "timebase_den: %d", voi->fps_num);
+	blog(LOG_INFO, "timebase_num: %d", voi->fps_den);
```
Comment (roddylindsay) [inline libobs/obs-encoder.c]: remove

File: libobs/obs-internal.h [hunk 4]
```diff
@@ -527,7 +527,7 @@ extern void obs_context_data_setname(struct obs_context_data *context,
 				     const char *name);
 
 /* ------------------------------------------------------------------------- */
-/* ref-counting  */
+/* ref-counting	 */
```
Comment (roddylindsay) [inline libobs/obs-internal.h]: revert

**Review by roddylindsay — CHANGES_REQUESTED** [review]

**Comment by RuiOrey** [comment]
removed the logs and wrong spacing on https://github.com/EnvelopSound/obs-studio/pull/1#discussion_r651305885

**Pull request merged**